cinema/views.py

```diff
@@ -34,6 +48,25 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    def get_queryset(self):
+        queryset = Movie.objects.all()
+
+        genres = self.request.query_params.get("genres", None)
+        if genres:
+            genres_ids = [int(str_id) for str_id in genres.split(",")]
```

Comment:
Consider adding error handling for the conversion of `genres` query parameter to integers. If a non-integer value is passed, it will raise a `ValueError`. You might want to handle this exception to avoid server errors.